spock-specs/src/test/groovy/org/spockframework/smoke/parameterization/DataTables.groovy

```diff
@@ -67,78 +63,96 @@ a
     1 | _
   }
 
-  def "table with just a header are not allowed"() {
+  def 'table with just a header are not allowed'() {
     when:
-    runner.runFeatureBody """
-expect:
-true
+    runner.runFeatureBody '''
+      expect:
+      true
 
-where:
-a | b
-    """
+      where:
+      a | b
+    '''
 
     then:
-    thrown(SpockExecutionException)
+    thrown SpockExecutionException
   }
 
-  def "header may only contain variable names"() {
+  def 'header may only contain variable names'() {
     when:
-    compiler.compileFeatureBody """
-expect:
-true
+    compiler.compileFeatureBody '''
+      expect:
+      true
 
-where:
-a | 1 | b
-1 | 1 | 1
-    """
+      where:
+      a | 1 | b
+      1 | 1 | 1
+    '''
 
     then:
-    thrown(InvalidSpecCompileException)
+    thrown InvalidSpecCompileException
   }
 
-  def "header variable names must not clash with local variables"() {
+  def 'header variable names must not clash with local variables'() {
     when:
-    compiler.compileFeatureBody """
-def local = 1
+    compiler.compileFeatureBody '''
+      def local = 1
 
-expect:
-true
+      expect:
+      true
 
-where:
-a | local
-1 | 1
-    """
+      where:
+      a | local
+      1 | 1
+    '''
 
     then:
-    thrown(InvalidSpecCompileException)
+    thrown InvalidSpecCompileException
   }
 
-    def "header variable names must not clash with each other"() {
+  def 'header variable names must not clash with each other'() {
     when:
-    compiler.compileFeatureBody """
-expect:
-true
+    compiler.compileFeatureBody '''
+      expect:
+      true
 
-where:
-a | a
-1 | 1
-    """
+      where:
+      a | a
+      1 | 1
+    '''
 
     then:
-    thrown(InvalidSpecCompileException)
+    thrown InvalidSpecCompileException
   }
 
-  def "columns can be declared as parameters"(a, String b) {
+  def 'columns can be declared as parameters'(a, String b) {
     expect:
     a == 3
-    b == "wow"
+    b == 'wow'
 
     where:
     a | b
-    3 | "wow"
+    3 | 'wow'
   }
 
-  def "pseudo-column can be omitted from parameter list"(a) {
```

Comment:
I tried separating it into commits for this exact reason. What do you suggest I do instead?